evennia/contrib/ingame_python/scripts.py

```diff
@@ -115,7 +115,11 @@ def get_events(self, obj):
         events = {}
         all_events = self.ndb.events
         classes = Queue()
-        classes.put(type(obj))
+        if isinstance(obj, ObjectDB):
+            classes.put(type(obj))
+        else:
+            classes.put(obj)
```

Comment:
I think a confusing error would be to put a Player typeclass in here. Maybe one could handle or warn about that more gracefully?